app/models/UcrBlock.scala

```diff
@@ -22,4 +22,7 @@ case class UcrBlock(ucr: String, ucrType: String)
 
 object UcrBlock {
   implicit val format = Json.format[UcrBlock]
+
+  val ducr = "D"
+  val mucr = "M"
```

Comment:
Maybe better call them `DucrType` and `MucrType` ?

Reply:
Can do.  And I don't think I've used them consistently - there are still some hard-coded "D" and "M" out there.

Doing this exercise highlighted how many times we have created constants/objects/enums to represent ducr or mucr in the product though!  :-)